.babelrc

```diff
@@ -1,14 +1,43 @@
 {
   "env": {
     "test": {
+      "presets": [
+        ["env", {"targets": { "node": "current" }}],
+        "react",
+        "stage-2"
+      ],
       "plugins": ["babel-plugin-rewire"]
     },
+    "development": {
+      "presets": [
+        ["env", { "targets": { "browsers": ["last 2 versions", "ie >= 9"] } }],
+        "react",
+        "stage-2"
```

Comment:
So if the features we use come from stage-2 and they are supported by the browsers of env then we should have stage-2 automatically applied, no?